plugins/azure-devops/README.md

```diff
@@ -49,6 +49,16 @@ spec:
   # ...
 ```
 
+#### Azure Pipelines Only
+
+If you are only using Azure Pipelines along with a different SCM tool then you can use the following annotation to see Builds:
+
+```yaml
+dev.azure.com/project-definition: <project-name>/<definition-name>
+```
+
+In this case `<project-name>` will be the name of your Team Project and `<definition-name>` will be the name of the Build Definition you would like to see Builds for. If the Build Definition name has spaces in it make sure to put quotes around it
```

Comment:
Naming things is hard. I guess I'm trying to convey that the first part of the annotation if the Project Name and the second part is the Build Definition Name. Does that make sense? With that is there something better you might suggest?

Reply:
Does it make more sense to split them apart if you can think of a nice name for a tuple?

Maybe something like:

```yaml
dev.azure.com/project: <project>
dev.azure.com/build-definition: <build-def>
```

Is arguably clearer and maybe less repetition (without knowing if the project and stuff is something you might re-use everywhere)

Reply:
That ^ or just `build-definition` since that seems to match what you'd call it in regular speech, if i understand things correctly. It's fine that it is defined to be a whole slug, in my opinion.

Reply:
Decided to go with:
```
dev.azure.com/project: <project>
dev.azure.com/build-definition: <build-def>
```

I think this will help us work towards supporting different organizations in the future